module_utils/storage_lsr/argument_validator.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import absolute_import, division, print_function
+
+import sys
+
+__metaclass__ = type
+
+# pylint: disable=undefined-variable
+STRING_TYPE = str if sys.version_info.major == 3 else basestring  # noqa:F821
```

Comment:
This is causing the ansible-test error:
```
Running sanity test "import" on Python 2.6
ERROR: Found 1 import issue(s) on python 2.6 which need to be resolved:
ERROR: plugins/module_utils/storage_lsr/argument_validator.py:8:0: traceback: AttributeError: 'tuple' object has no attribute 'major'
```
I guess `sys.version_info` is just a plain tuple on python 2.6

The solution is to add the following line to all of the .sanity-ignore*.txt files:
```
plugins/module_utils/storage_lsr/argument_validator.py import-2.6!skip
```